pulsar-client-cpp/include/pulsar/ProducerConfiguration.h

```diff
@@ -164,13 +204,52 @@ class PULSAR_PUBLIC ProducerConfiguration {
      */
     int getMaxPendingMessagesAcrossPartitions() const;
 
+    /**
+     * Set the message routing modes for partitioned topics.
+     *
+     * @param PartitionsRoutingMode partition routing mode.
+     * @return
+     */
     ProducerConfiguration& setPartitionsRoutingMode(const PartitionsRoutingMode& mode);
+
+    /**
+     * The getter associated with setPartitionsRoutingMode().
+     */
     PartitionsRoutingMode getPartitionsRoutingMode() const;
 
+    /**
+     * Set a custom message routing policy by passing an implementation of MessageRouter.
+     *
+     * @param messageRouter message router.
+     * @return
+     */
     ProducerConfiguration& setMessageRouter(const MessageRoutingPolicyPtr& router);
+
+    /**
+     * The getter associated with setMessageRouter().
+     */
     const MessageRoutingPolicyPtr& getMessageRouterPtr() const;
 
+    /** Set the hashing scheme, which is a standard hashing function available when choosing the partition to
+     * use for a particular message.
```

Comment:
```suggestion
    /**
     * Set the hashing scheme, which is a standard hashing function available when choosing the partition to
     * use for a particular message.
```

Reply:
@BewareMyPower thanks for your comments, I've incorporated them, PTAL.